### PR TITLE
Add video-miner to community

### LINKED ISSUE #189: Add Video Miner to the /content/community section of https://livepeer.org/ecosystem
Video Miner is a Livepeer pool and we'd like to have it listed on the ecosystem page. We've submitted an application through the page but haven't heard anything back in a while. I'm hoping someone can pick this us and get us listed :) 

If any additional information is needed from our side, please let us know.

### PR DESCRIPTION
**Pull request by eliteprox**
Adds Video-Miner to the Livepeer ecosystem page.
This PR will close #189

### CODE REVIEW

**Review by adamsoffer — CHANGES_REQUESTED** [review]
Thanks for the PR @eliteprox. Can you please move this into the `community` directory?

https://github.com/livepeer/org/tree/main/content/community

**Comment by eliteprox** [comment]
> Thanks for the PR @eliteprox. Can you please move this into the `community` directory?
> 
> https://github.com/livepeer/org/tree/main/content/community

Thanks, I have moved the file now

File: content/community/video_miner.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+publish: true
+image: >-
+  https://user-images.githubusercontent.com/16746274/222543634-6ffa38ad-d692-4dee-b800-1d969ad6dec7.png
```
Comment (eliteprox) [inline content/community/video_miner.md:4]: Please upload the attached logo to cloudinary and update the image url.
videominer_logo_ecosystem2-01-02

**Comment by AuthorityNull** [comment]
Hi @adamsoffer, can we get an update on this? Thanks.

**Review by adamsoffer — APPROVED** [review]

**Comment by eliteprox** [comment]
The image still needs to be uploaded to cloudinary. Core team will handle that, correct?

**Pull request merged**